packages/swapper/src/swappers/JupiterSwapper/endpoints.ts

```diff
@@ -45,6 +46,54 @@ export const jupiterApi: SwapperApi = {
 
     return (await adapter.buildSendApiTransaction(buildSwapTxInput)).txToSign
   },
+  getSolanaTransactionFees: async ({
+    tradeQuote,
+    from,
+    assertGetSolanaChainAdapter,
+  }: GetUnsignedSolanaTransactionArgs): Promise<string> => {
+    if (!isExecutableTradeQuote(tradeQuote)) throw Error('Unable to execute trade')
+
+    const step = tradeQuote.steps[0]
+
+    const adapter = assertGetSolanaChainAdapter(step.sellAsset.chainId)
+
+    if (!isExecutableTradeStep(step)) throw Error('Unable to execute step')
+
+    const solanaInstructions = step.jupiterTransactionMetadata?.instructions?.map(instruction =>
+      adapter.convertInstruction(instruction),
+    )
+
+    if (!isSolanaFeeData(step.feeData.chainSpecific)) throw Error('Unable to execute step')
+
+    const buildSwapTxInput: BuildSendApiTxInput<KnownChainIds.SolanaMainnet> = {
+      to: '',
```

Comment:
Looks odd but is correct, same payload as `getUnsignedSolanaTransaction`, `to` gets added in `adapter.buildSendApiTransaction()` below